Workflow/Sources/SubtreeManager.swift

```diff
@@ -272,7 +272,7 @@ extension WorkflowNode.SubtreeManager {
 
     fileprivate final class ReusableSink<Action: WorkflowAction>: AnyReusableSink where Action.WorkflowType == WorkflowType {
         func handle(action: Action) {
-            let output = Output.update(AnyWorkflowAction(action), source: .external)
+            let output = Output.update(action, source: .external)
```

Comment:
this change will help the [observation API](https://github.com/square/workflow-swift/pull/168) be more useful as we will be able to expose the underlying conformance and not an AnyWorkflow wrapper